pkg/secretlock/secretlock.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
just curious why this is a second file?

Reply:
to avoid mixing structs, vars or constants with API interfaces...